docker/worker/worker.py

```diff
@@ -316,6 +316,16 @@ def maybe_normalize_package_names(vulnerability):
   return vulnerability
 
 
+def filter_unsupported_ecosystems(vulnerability):
+  """Remove unsupported ecosystems from vulnerability"""
```

Comment:
Nit: use periods at the end of sentences. https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings